app/models/post.rb

```diff
@@ -0,0 +1,8 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  after_create :update_user_posts_counter
+
```

Comment:
Hi there,

I've gone through your `Post` model implementation, and I see that you have set up some of your associations properly. However, it looks like we're missing an instance method that is required as per the project specifications. Each post should be able to return its five most recent comments. This functionality is crucial for the feature that allows users to view the latest discussions on your platform quickly.


### Common Example of the method
```ruby
class Post < ApplicationRecord
  # ... existing code ...
  
  def recent_comments
    comments.order(created_at: :desc).limit(5)
  end
end
```

This `recent_comments` method will sort the comments associated with a post in descending order, ensuring the newest comments are returned first, and then it will limit the results to the five most recent entries.

Please implement this method in your `Post` model, and don't hesitate to reach out if you need any further clarification or assistance.

Best,
Temitope

---